src/uct/cuda/cuda_ipc/cuda_ipc_cache.h

```diff
@@ -26,6 +26,7 @@ struct uct_cuda_ipc_cache_region {
     ucs_list_link_t         list;         /**< List element */
     uct_cuda_ipc_key_t      key;          /**< Remote memory key */
     void                    *mapped_addr; /**< Local mapped address */
+    int                     refcount;     /**< Track inflight ops before unmapping*/
```

Comment:
Maybe better to keep it unsigned, but check for
(region->refcount >= 1) or for (region->refcount != 0) before doing region->refcount— ? And (region->refcount < UINT_MAX) before doing region->refcount++ ?

Reply:
Thanks @dmitrygx I've incorporated your suggestions.